src/components/data/common/Pagination/Pagination.tsx

```diff
@@ -1,16 +1,47 @@
-import React, { Component } from 'react';
+import React, { Component, ReactElement } from 'react';
 import PropTypes from 'prop-types';
-import _ from 'lodash';
+import { isEmpty } from 'lodash';
 
 import { Icon, Grid, Message, Pagination as PaginationNavigation } from 'semantic-ui-react';
 import Popup from 'components/popups/Popup';
 import PaginationInfo from './PaginationInfo';
 
 import './Pagination.css';
 
-export default class Pagination extends Component {
-    constructor(props, context) {
-        super(props, context);
+interface PaginationProps {
+    fetchData: () => void;
+    totalSize: number;
+    pageSize: number;
+    sizeMultiplier: number;
```

Comment:
Since they are optional shouldn't we define them this way:
```suggestion
    totalSize?: number;
    pageSize?: number;
    sizeMultiplier?: number;
```
?

Reply:
The answer is the same as for https://github.com/cloudify-cosmo/cloudify-ui-components/pull/76#discussion_r677256691 :slightly_smiling_face:

Reply:
👍🏻